app/views/admin/enterprises/form/_business_address.html.haml

```diff
@@ -0,0 +1,41 @@
+.row
+  .three.columns.alpha
+    = bf.label :company, t(".company_legal_name")
+  .eight.columns.omega
+    = bf.text_field :company, { placeholder: t(".company_placeholder") }
+
+.row
+  .three.columns.alpha
+    = bf.label :address1, t('.address1')
+  .eight.columns.omega
+    = bf.text_field :address1, { placeholder: t(".address1_placeholder") }
+.row
+  .alpha.three.columns
+    = bf.label :address2, t(".address2")
+  .eight.columns.omega
+    = bf.text_field :address2
+.row
+  .three.columns.alpha
+    = bf.label :city, t(:city)
+    \/
+    = bf.label :zipcode, t(:postcode)
+  .four.columns
+    = bf.text_field :city, { placeholder: t(:city_placeholder) }
+  .four.columns.omega
+    = bf.text_field :zipcode, { placeholder: t(:postcode_placeholder) }
+.row
+  .three.columns.alpha
+    = bf.label :state_id, t(:state)
+    \/
+    = bf.label :country_id, t(:country)
+  %div{ "ng-controller": "countryCtrl", "ng-init": "address_type='business_address'" }
+    .four.columns
+      %input.ofn-select2.fullwidth#enterprise_business_address_attributes_state_id{ name: 'enterprise[business_address_attributes][state_id]', type: 'number', data: 'countriesById[Enterprise.address.country_id].states', placeholder: t('admin.choose'), ng: { model: 'Enterprise.business_address.state_id' } }
+    .four.columns.omega{ "ng-controller" => "countryCtrl" }
```

Comment:
Ah, we can drop this bit as well now that the controller is on the parent div wrapping both elements.
```suggestion
    .four.columns.omega
```